public/js/stats.js

```diff
@@ -8,14 +8,14 @@ $(document).ready(function() {
 
     overviewSections = {
         "total-records-over-time": {
-            url: Routing.generate("pelagos_app_ui_stats_getdatasetovertime"),
+            url: "stats/data/total-records-over-time",
```

Comment:
Use route instead.

Reply:
Yeah, I think this breaks when I deploy it onto the server without the route. The pelagos-symfony part of the route gets eaten.